src/derivatives/jp2/make_jp2.go

```diff
@@ -156,52 +155,11 @@ func (t *Transformer) makeJP2() {
 	// floats really aren't)
 	var baseRate = int(t.getRate() * RateFactor)
 
-	var rangeQueue = &RangeQueue{}
-	t.testedRates = make(map[int]bool)
-
-	// First "range" is just the base rate since that's the ideal value
-	rangeQueue.Append(baseRate, baseRate)
-
-	// 2/3 * x to try something semi-distant but not super expensive in terms of storage
-	rangeQueue.Append(baseRate*2/3, baseRate)
-
-	for i := 0; len(rangeQueue.queue) > 0; i++ {
-		var r = rangeQueue.Shift()
-		if r == EmptyRange {
-			continue
-		}
-
-		if t.testRate(r.start) {
-			return
-		}
-		if t.testRate(r.end) {
-			return
-		}
-
-		var midPoint = (r.start + r.end) / 2
-		rangeQueue.Append(r.start, midPoint)
-		rangeQueue.Append(midPoint, r.end)
-
-		// After a while we are willing to lose a little quality
-		if i == 5 {
-			rangeQueue.Append(baseRate, baseRate*5/4)
-		}
-
-		// Later on, we expand the search further.  This can cost a good deal of
-		// space, but we're getting desperate.
-		if i == 50 {
-			rangeQueue.Append(baseRate/3, baseRate*2/3)
-		}
-
-		// If we had no successes after exhausting all those options above, we're
-		// willing to sacrifice a lot more quality or space
-		if i > 50 && len(rangeQueue.queue) == 0 {
-			rangeQueue.Append(baseRate/6, baseRate/3)
-			rangeQueue.Append(baseRate*5/4, baseRate*3/2)
-		}
+	if t.testRate(baseRate) {
+		return
 	}
 
-	t.err = fmt.Errorf("no rate found for creating a valid JP2")
+	t.err = fmt.Errorf("default rate does not create a valid JP2")
```

Comment:
Since rates are no longer a variable factor, I'd prefer the error value not mention it at all anymore.  e.g., "couldn't create a valid JP2"